code/models/ElementVirtualLinked.php

```diff
@@ -46,7 +46,7 @@ public function __construct($record = null, $isSingleton = false, $model = null)
     public function getCMSFields()
     {
         $message = sprintf(
-            '<p>%s</p><p><a href="%2$s">%2$s</a></p>',
+            '<p>%s</p><p><a href="%2$s" target="_blank">%2$s</a></p>',
```

Comment:
I doubt this'll ever be tabnab-able but do you want to add in `rel="noopener noreferrer"` just in case?